lib/autsr.gi

```diff
@@ -170,10 +170,74 @@ end);
 
 # step through intermediate subgroups to find subgroup that satisfies
 # condition
-BindGlobal("SubgroupConditionAbove",function(G,cond,S)
-local c,hom,q,a,t,int,bad,have,ups,up,new,u;
+InstallGlobalFunction(SubgroupConditionAbove,function(G,cond,S)
+local c,hom,q,a,b,i,t,int,bad,have,ups,up,new,u,good;
+
+  Info(InfoMorph,2,"SubgroupAbove ",IndexNC(G,S));
+
+  # first try, how far `SubgroupProperty` goes
+  good:=[];
+  bad:=[];
+  b:=0;
+  a:=false;
```

Comment:
could perhaps `a` be renamed to `abort` or `bailout` or something?

Reply:
But then I'm using a further variable (since `a` is re-used later) :-)
(Yes, I've never made the step of realizing that I have more than 8 registers to work with). Changed.

Reply:
Heh